src/software/visualizer/widgets/ai_control.cpp

```diff
@@ -86,21 +86,21 @@ void setupGameStateOverrideComboBox(Ui::AutoGeneratedMainWidget *widget)
     widget->gamestate_override_combo_box->insertItem(2, "Halt");
     widget->gamestate_override_combo_box->insertItem(3, "Stop");
 
-    auto on_gamestate_changed = [](const QString &text) {
+    auto on_gamestate_changed = [config](const QString &text) {
         if (text == "Use Refbox")
         {
-            Util::MutableDynamicParameters->getMutableAIControlConfig()
+            config->getMutableAIControlConfig()
                 ->getMutableRefboxConfig()
                 ->mutableOverrideRefboxPlay()
                 ->setValue(false);
         }
         else
         {
-            Util::MutableDynamicParameters->getMutableAIControlConfig()
+            config->getMutableAIControlConfig()
                 ->getMutableRefboxConfig()
                 ->mutableOverrideRefboxPlay()
                 ->setValue(true);
-            Util::MutableDynamicParameters->getMutableAIControlConfig()
+            config->getMutableAIControlConfig()
                 ->getMutableRefboxConfig()
                 ->mutableCurrentRefboxPlay()
                 ->setValue(text.toStdString());
```

Comment:
We could probably clean this up a bit by adding `auto refbox_config = config->getMutableAIControlConfig()->getMutableRefboxConfig()->getMutableRefboxConfig();`  and then using that, but totally your call, it's not too bad as-is.

Reply:
done

Reply:
Just realized there's one more case of this above around line 60. Can we change that as well to be consistent?

Reply:
changed